paddle/fluid/framework/ir/graph.h

```diff
@@ -47,6 +47,10 @@ constexpr char kStaleProgramOpDescs[] = "stale_program_op_descs";
 
 namespace ir {
 
+const char kGraphToProgramVarsToRemove[] =
```

Comment:
Should we put it at `graph.h`? It is only used in GraphToProgram and your PR moved the GraphToProgram to graph_helper, can we put those constant at `graph_helper.h`?

Reply:
Done.